File: 01 - Nivelamento/Exercicios/unidade00o_processoAEDIIExercicios/10 - Cometa em C-C++/Cometa.c
#include <stdio.h>

int main() {
    int year, quantidadeDeVezes;

    while (year != 0) {
        scanf("%d", &year);

        quantidadeDeVezes = (int) (year - 10) / 76;
        quantidadeDeVezes = quantidadeDeVezes + 1;

        if (year != 0) {
            printf("%d\n", quantidadeDeVezes * 76 + 10);
        }
    }

    return 0;
}

File: Trabalhos Teoricos/TrabalhoTeorico2/09 - Cometa em Java/Cometa.java
public class Cometa {
    public static void main(String[] args) {
        int year = -1, quantidadeDeVezes = 0;

        while (year != 0) {
            year = MyIO.readInt();

            quantidadeDeVezes = (year - 10) / 76;
            quantidadeDeVezes = quantidadeDeVezes + 1;

            if (year != 0) {
                MyIO.println(quantidadeDeVezes * 76 + 10);
            }
        }
    }
}
